sdk-java/src/main/java/ly/count/sdk/java/Config.java

```diff
@@ -656,6 +666,23 @@ public Config setUsePOST(boolean usePOST) {
         return this;
     }
 
+    /**
+     * Enable SDK's backend mode.
+     * @return {@code this} instance for method chaining
+     */
+    public Config enableBackendMode() {
+        this.enableBackendMode = true;
+        return this;
+    }
+
+    public int getRequestQueueMaxSize() {
+        return requestQueueMaxSize;
+    }
+
+    public void setRequestQueueMaxSize(int requestQueueMaxSize) {
+        this.requestQueueMaxSize = requestQueueMaxSize;
```

Comment:
During init we need to verify that this is not less then 1. If it is, set it to 1.